src/webapp/src/auth/AuthenticationOptions.tsx

```diff
@@ -1,7 +1,68 @@
+import { Button } from '@small-ads/ui';
 import React from 'react';
+import styled from 'styled-components';
+import { FacebookSquare } from '@styled-icons/boxicons-logos';
+
+const LoginScreen = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: calc(100vh - 64px);
+`;
+
+const LoginMain = styled.div`
+  align-self: center;
+  flex: 1 1;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+`;
+
+const LoginSign = styled.div`
+  font-size: 3rem;
+  letter-spacing: -0.066875rem;
+  font-weight: 700;
+`;
+
+const LoginForm = styled.div`
+  width: 100%;
+  text-align: center;
+  max-width: 320px;
+  align-self: center;
+`;
+
+const GeistSpacer = styled.span`
+  margin-left: 23px;
+  margin-top: 23px;
+`;
+
+const FbButton = styled(Button)`
+  &&& {
```

Comment:
Never seen this before, what does it do?

Reply:
We need it to increment the [specificity](https://styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity) of the `class-name` inserted by `styled-components`, if we don't do it the styles are going to be overridden by those from the component we are importing from the `ui`

Reply:
Cool stuff! Always learning something new with you!